### DISCUSSION TITLE
USB Tethering

### DISCUSSION THREAD

**Discussion by andrewclo** [SmoothWAN/SmoothWAN#81] [General & Help]
Hi, 

Just installed SmoothWAN on a Slate router. Working great so far. Is it possible to use the USB port on the Slate for USB tethering for a cell phone data connection?

**Comment by TalalMash** [comment]
It should be automatic with Android phones, iOS 13+ is tricky to get it working, I can't test the latest versions since I don't have a recent device. There is more info in OpenWRT wikis.

**Reply by TalalMash** [reply]
You can check if the device is detected in _Network -> Devices_ tab , it should be `usb0`

**Reply by andrewclo** [reply]
Thanks for the quick reply. It doesn't look the router can see the USB connection from what I can tell. 
<img width="1688" alt="Screenshot 2022-11-29 at 5 59 48 PM" src="https://user-images.githubusercontent.com/1765900/204689174-03613762-a9f7-40eb-b870-b09d1f9dafb1.png">
<img width="1688" alt="Screenshot 2022-11-29 at 5 59 45 PM" src="https://user-images.githubusercontent.com/1765900/204689178-259ea5c0-5020-4914-9b3d-40ca9960a313.png">

**Reply by TalalMash** [reply]
Thanks for the tip! :)
I can see iPhone in devices, you can switch back to Interfaces and click edit next to AUTOUSB1 and switch from `usb0` to `iphone`.
(This is the first time I see the name as iphone, the usbmuxd app handles parameters from iOS., will add next release) 
Edit: Turned out that was a wireless connection (?)

**Reply by TalalMash** [reply]
Slate AX current kernel has issues with USB devices, fixed in next release 0.99.9
